### PR TITLE
Unify serverro role partition config with Kubic (boo#1087449)

### PR DESCRIPTION
**Pull request by sysrich**

### CODE REVIEW

**Comment by okurz** [comment]
In the commit message, serverro role ? Should it be "server role" or is there something missing?

File: control/control.openSUSE.xml [hunk 1]
```diff
@@ -505,19 +505,21 @@ textdomain="control"
...
+                    <min_size config:type="disksize">16 GiB</min_size>
+                    <max_size config:type="disksize">30 GiB</max_size>
+                    <weight config:type="integer">80</weight>
 
                     <snapshots config:type="boolean">true</snapshots>
-                    <snapshots_configurable config:type="boolean">true</snapshots_configurable>
-                    <snapshots_percentage config:type="integer">100</snapshots_percentage>
+                    <snapshots_configurable config:type="boolean">false</snapshots_configurable>
+
+                    <!-- You don't want to miss the / volume -->
```
Comment (jreidinger) [inline control/control.openSUSE.xml]: I do not get this comment, can you be more specific what is problem?

Comment (sysrich) [reply control/control.openSUSE.xml]: For the filesystem on a transactional rootfs you really need to make sure the root filesystem is configured exactly as proposed. The comment is actually copied from the same lines of config from skelcd-control-CAASP, as this unification applies to all 3 systems using transactional hosts (CaaSP, Kubic, and now here)

Comment (jreidinger) [reply control/control.openSUSE.xml]: ok, so at least it is not clear everywhere :)

**Pull request merged**